setup.py

```diff
@@ -0,0 +1,49 @@
+#   Copyright 2019-2020 Honeywell, Intl. (www.honeywell.com)
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import setuptools
+import os
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+requirements = [
+    'nest-asyncio>=1.2.0',
+    'qiskit-terra==0.8',
```

Comment:
Is this a hard requirement? 0.8 is an old release almost a year old at this point and because of changes made to the ibmq provider and terra since I think this will be an issue for many qiskit users.